packages/carbon-graphs/CHANGELOG.md

```diff
@@ -2,4 +2,7 @@
 
 ## Unreleased
 
+* Changed
+  * Added code to handle null/undefined/blank in paired result graph during both initial load and reflow.
+  
 -   Migrated to the terra-graphs mono-rep.
```

Comment:
Nit: Have consistent syntax for bullet points: use `*` or `-`

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/19/commits/9a39a0621365a28bebb30efc458d8eac9949b4bf#diff-7b73f4b73c95577a16e8a58d2e1377b2204811d315c020884e80b9f22128c42aR6).